pyproject.toml

```diff
@@ -54,23 +54,7 @@ markers = [
 ]
 filterwarnings = [
     "error",
-    "default:Error (reading|writing) persistent compilation cache entry for 'jit_equal'",
-    "default:Error (reading|writing) persistent compilation cache entry for 'jit__lambda_'",
-    "default:jax.extend.mlir.dialects.mhlo is deprecated.*:DeprecationWarning",
 
-    # TODO(jakevdp): remove when array_api_tests stabilize
-    "default:.*not machine-readable.*:UserWarning",
-    "default:Special cases found for .* but none were parsed.*:UserWarning",
-    "default:.*is not JSON-serializable. Using the repr instead.*:UserWarning",
-    "default:The .* method is good for exploring strategies.*",
-
-    # These are transitive warnings coming from TensorFlow dependencies.
-    # TODO(slebedev): Remove once we bump the minimum TensorFlow version.
-    "default:The key path API is deprecated .*",
-    "default:jax.xla_computation is deprecated.*:DeprecationWarning",
```

Comment:
I know that the current tensorflow release (2.17) still emits this warning, so this will probably break the jax2tf test.

Reply:
Oh, looks like this is fine after all!